Apps/AdminWebClient/src/ClientApp/src/services/restConfigService.ts

```diff
@@ -23,4 +25,13 @@ export class RestConfigService implements IConfigService {
                 });
         });
     }
+    public getUserProfilesExportUrl(): string {
```

Comment:
The vuex store has a getter to get the configuration. For this case in particular, there is a serviceEndpoints getter that can be used.
```
export const getters: GetterTree<ConfigState, RootState> = {
    serviceEndpoints(state: ConfigState): { [id: string]: string } {
        const { config } = state;
        return config.serviceEndpoints;
    },
    openIdConnect(state: ConfigState): OpenIdConnectConfiguration {
        const { config } = state;
        const { openIdConnect } = config;
        return openIdConnect;
    },
    admin(state: ConfigState): AdminClientConfiguration {
        const { config } = state;
        const { admin } = config;
        return admin;
    }
};
```

Let me know if you need help with the vuex store. There are examples with the annotation of @Getter

Reply:
nice one. I think I can update code based on vuex store.